Flick/Models/Media.swift

```diff
@@ -10,26 +10,26 @@ import Foundation
 
 struct SimpleMedia: Codable {
     var id: Int
-    var posterPic: String
+    var posterPic: String?
 }
 
 struct Media: Codable {
     var id: Int
     var title: String
-    var posterPic: String
-    var directors: String
+    var posterPic: String?
```

Comment:
lol what are we gonna do if poster pic comes back nil... we should talk to backend about this